content/applications/productivity/iot/config/troubleshooting.rst

```diff
@@ -5,130 +5,153 @@ Troubleshooting
 IoT Box Connection
 ==================
 
-I can't find the pairing code to connect my IoT Box
----------------------------------------------------
+Unable to locate the pairing code to connect the IoT Box
+--------------------------------------------------------
 
-The pairing code should be printed on receipt printers connected to the
-IoT Box and should also be displayed on connected monitors.
+The pairing code should be printed on receipt printers connected to the IoT Box and should also be
+displayed on connected monitors.
 
 The pairing code doesn't show under the following circumstances:
 
 -  The IoT Box is already connected to an Odoo database;
 
 -  The IoT Box is not connected to the Internet;
 
--  The code is only valid for 5 minutes after the IoT Box has started. It's
-   automatically removed from connected displays when this delay has expired;
+-  The code is only valid for 5 minutes after the IoT Box has started. It's automatically removed
+   from connected displays when this delay has expired;
 
--  The version of the IoT Box image is too old. It should use version 20.06
-   or more recent. If your IoT Box image is from an earlier version, then
-   you will have to reflash the SD card of your IoT Box to update the image
-   (see :doc:`Flashing your SD Card <flash_sdcard>`)
+-  The version of the IoT Box image is too old. It should use version 20.06 or more recent. If the
+   IoT Box image is from an earlier version, then the SD card of the IoT Box will need to be
+   re-flashed to update the image. (see :doc:`Flashing the SD Card <flash_sdcard>`)
 
-If you are not in any of the cases listed above, make sure that the IoT Box
-has correctly started, by checking that a fixed green LED is showing next to
-the micro-USB port.
+If none of the cases listed above correct the issue, then make sure that the IoT Box has correctly
+started, by checking that a fixed green LED is showing next to the micro-USB port.
```

Comment:
> next to the micro-USB port
"next to the power port" (as micro usb is not used on raspberri pi model 4)

Reply:
That is another great suggestion. Thanks! 👍